scss/styles.scss

```diff
@@ -0,0 +1,296 @@
+@import "_reset";
+@import "_mixins";
+@import "_fonts";
+
+//variables
+$main-gray: #626262;
+$support_gray: #f3f3f3;
+$aside: #b4ad9e;
+
+
+$firstBreakPoint: 600px;
+$secondBreakPoint: 900px;
+$thirdBreakPoint: 1200px;
+
+
+
+
+body{
+  font-family: Playfair, Arial, sans-serif;
+  font-size: 16px;
+}
+.main-content-wrapper{
+  padding-top: 56px;
+}
+
+.wrapper{
```

Comment:
mógłbyś też nadać mu `padding-left, padding-right`. W tej chwili nadajesz `margin-left, margin-right` artykułom, żeby odsunąć je od brzegów strony. To jest zawodne rozwiązanie, bo musisz pamietać, żeby wszystkim elementom nadawać odpowiednie `margin` jeśli chcesz żeby były w równym odstępie od krawędzi i jeśli coś poprzestawiasz w layoucie to może Ci się zwyczajnie posypać. Lepiej byłoby nadać `.wrapper` odpowiedni `padding`

Reply:
nie rozumiem